docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,156 @@
+
+[[bxms_rn_whats_new]]
+= What's New
+
+This section highlights new features in {PRODUCT} {PRODUCT_VERSION} Limited Availability.
+
+== Business Central
+ 
+=== Improved Navigation
+
+.Navigation between views
+You can now navigate as follows:
```

Comment:
Really? I don't see a rule about 'as follows' in the IBMSG, but I find 13 examples of it's use, so I assume that by example IBM is OK with it:
Explain the parameters and their options after the syntax diagram, as follows:
In the glossary, specify your selected terms as follows:
and so on.

Reply:
OK, may have changed since I learned the ropes there in 2008-9. I am still waiting on my copy so was going from memory. "The following:" without a noun is more definitely a no-no. Let's keep "as follows" then.